purchases/src/customEntitlementComputation/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -277,6 +277,7 @@ class Purchases internal constructor(
             val configuration = PurchasesConfiguration.Builder(context, apiKey)
                 .appUserID(appUserID)
                 .dangerousSettings(DangerousSettings(customEntitlementComputation = true))
+                .pendingTransactionsForPrepaidPlansEnabled(true)
```

Comment:
why is this always true?

Reply:
I believe we want to enable it by default for companies using `customEntitlementComputation`. We can provide an option to disable it later if needed though, but we try to keep the API very small in the `customEntitlementComputation` flavor I think.